video_xblock/video_xblock.py

```diff
@@ -761,12 +761,20 @@ def update_metadata_authentication(self, auth_data, player):
             self.metadata['client_secret'] = ''  # Brightcove API
 
     @XBlock.handler
-    def upload_manual_transcript_handler(self, request, suffix=''):
+    def upload_file_handler(self, request, suffix=''):
         """
-        Upload manual transcript for video xblock.
+        Upload manual transcript or handouts for video xblock.
 
         Saves transcript as the .vtt file.
-        Performs file conversion beforehand (if a different supported file type is received).
+        Performs file conversion beforehand, from srt to vtt
+        Saves handouts as:
+            images: .gif, .ico, .jpg, .jpeg, .png, .tif, .tiff, .bmp, .svg
+            documents: .pdf, .txt, .rtf, .csv
+            MSOffice: .doc, .docx, .xls, .xlsx, .ppt, .pptx, .pub
+            openOffice: .odt, .ods, .odp
+            archives: .zip, .7z, .gzip, .tar
+            other: .html, .xml, .js, .sjson
+            transcripts: .vtt, .srt (converted to the .vtt format)
```

Comment:
This is not the responsibility nor the requirement of the handler. It relies on the frontend and should be removed from here.

Reply:
done